distrho/src/DistrhoUIVST3.cpp

```diff
@@ -538,6 +556,14 @@ class UIVst3
     bool fIsResizingFromPlugin;
     bool fIsResizingFromHost;
 
+    // Native timer support
```

Comment:
these should be under a `#if DISTRHO_PLUGIN_HAS_EXTERNAL_UI` as they are unused otherwise

Reply:
Are you sure these are not used by the Pugl-based implementation?